rackcity/utils/change_planner_utils.py

```diff
@@ -102,6 +102,8 @@ def detect_conflicts_cp(sender, **kwargs):
 
 
 def get_change_plan(change_plan_id):
+    if change_plan_id is None:
+        return None, None
```

Comment:
added this to the helper method because it had the same exact line of code before it everywhere it was being used